iothub/service/src/Amqp/AmqpTransportLog.cs

```diff
@@ -13,9 +13,7 @@ internal class AmqpTransportLog : AmqpTrace
         public override void AmqpAbortThrowingException(string exception)
         {
             if (Logging.IsEnabled)
-            {
```

Comment:
We've chosen to omit curly braces for logging statements to avoid logging taking up quite so much room in our files.